proxies/live/apiproxy/resources/jsc/ConnectionMetadata.SetRequestHeaders.js

```diff
@@ -24,14 +24,14 @@
     // Read the 'NHSD-End-User-Organisation-ODS' header
     var odsCode = context.getVariable('request.header.NHSD-End-User-Organisation-ODS');
     if (!odsCode || odsCode.trim().length === 0) {
-        context.setVariable("badRequest", true);
+        //This will trigger RaiseFault.400BadRequest.xml - see proxies/deafult.xml in the DefaultFaultRules
```

Comment:
Comments on the PR need to point to RaiseFault.**[403NoPointers.xml]** instead of RaiseFault.**400BadRequest.xml**

Reply:
Ah this comment is correct, but the one below does need to change :)